addons/ofxOsc/src/ofxOscMessage.cpp

```diff
@@ -237,12 +278,16 @@ ofxOscMessage& ofxOscMessage::copy( const ofxOscMessage& other )
 	for ( int i=0; i<(int)other.args.size(); ++i )
 	{
 		ofxOscArgType argType = other.getArgType( i );
-		if ( argType == OFXOSC_TYPE_INT32 )
```

Comment:
not really the submitter's fault, but why on earth is this not a switch statement instead of if/else tree? we're testing for enums, after all...